src/components/user-level.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import React, { useState } from 'react';
+
+import { GraphElements } from '@/components/ui/graphElements';
+
+import Widget from './ui/widget';
+
+interface ScoreProps {
+  score: number;
+  level: number;
+}
+
+const UserLevel = (scoreProps: ScoreProps) => {
+  const [score] = useState<number>(scoreProps.score);
```

Comment:
Just so you'll know, but later on we'll probably need to retrieve this from zustand state